generic/samples/youkube/src/Database.kt

```diff
@@ -1,11 +1,11 @@
 package io.ktor.samples.youkube
 
-import com.google.gson.*
-import org.ehcache.*
-import org.ehcache.config.*
-import java.io.*
-import java.util.*
-import java.util.concurrent.atomic.*
+import com.google.gson.GsonBuilder
```

Comment:
Consider using `*` imports here

Reply:
It's auto formatted, so I prefer to leave it as it is. We can update checkstyle later.

Reply:
It's because of the IDE config. Please consider enabling `*` imports in the autoformater